styles.css

```diff
@@ -67,10 +71,377 @@ body {
   margin-bottom: 12px;
 }
 
+.connect a {
+  text-decoration: none;
+  color: blue;
+}
+
 .links ul {
   list-style-type: none;
   display: flex;
   gap: 20px;
   padding: 0;
   padding-bottom: 138px;
 }
+
+/* mid section */
+
+#mid {
+  display: grid;
+  grid-template-columns: 24px 1fr 24px;
+  row-gap: 88px;
+  background-color: #fff;
+}
+
+.left-block {
+  grid-column: 2;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #def1e6;
+}
+
+.primary-image {
+  border-radius: 8px;
+  margin-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 12px;
+}
+
+.title h1 {
+  margin-bottom: 20px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.frame-2 {
+  margin-bottom: 20px;
+  margin-left: 16px;
+}
+
+.frame-2 ul {
+  display: flex;
+  padding: 0;
+}
+
+.frame-2 ul li {
+  margin-right: 32px;
+}
+
+.primary-text {
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 16px;
+  max-width: 295px;
+  max-height: 76px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.tags {
+  margin-bottom: 40px;
+  margin-left: 0;
+  color: #6070ff;
+}
+
+.tags ul {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  margin-left: 16px;
+}
+
+.tags ul li {
+  line-height: 16px;
+  letter-spacing: 3%;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.action {
+  margin-left: 16px;
+  margin-bottom: 40px;
+  border: solid #6070ff;
+  width: 124px;
+  height: 48px;
+  padding: 12px;
+  font-weight: 500;
+  border-radius: 8px;
+  color: #396df2;
+}
+
+.action a {
+  color: #396df2;
+  text-decoration: none;
+  border-radius: 8px;
+}
+
+/* About section */
+
+.upper-part {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.about-myself {
+  max-width: 327px;
+  max-height: 101px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 12px;
+  margin-top: 144px;
+}
+
+.about-text {
+  max-width: 327px;
+  max-height: 144px;
+  font-size: 16px;
+  margin-bottom: 12px;
+}
+
+.about-connect {
+  margin-bottom: 16px;
+  color: #7f8cff;
+}
+
+.about-connect a {
+  text-decoration: none;
+  color: #7f8cff;
+}
+
+.about-tags ul {
+  display: flex;
+  list-style: none;
+  gap: 21px;
+  padding-left: 11px;
+}
+
+.resume {
+  height: 48px;
+  width: 162px;
+  left: 375px;
+  top: 4769px;
+  border-radius: 8px;
+  margin-top: 34px;
+  padding: 12px;
+  color: #396df2;
+  margin-bottom: 60px;
+  border: 1px solid #6070ff;
+}
+
+.resume a {
+  text-decoration: none;
+  color: #396df2;
+}
+
+.action a:hover {
+  color: orange;
+  background-color: #396df2;
+}
```

Comment:
- [ ] I think that the hover effect should be on the `div.action` class instead of the `a` because currently, the hover effect appears like this 👇. 
![image](https://user-images.githubusercontent.com/84156714/193411329-94d008ea-81f6-426e-94dc-ea9d95daae03.png)

Instead, it should cover the entire `See project button` like this 👇 
![](https://user-images.githubusercontent.com/66992901/193404080-3fedfa1b-ed9c-4879-a0f0-000bc3101cb8.PNG) 

Alternatively, you could use a `button` element with class name `action` -> like this `<button type"button" class="action">See project</button>` , then apply the hover effect on the button. This will mean that you will remove both the `div` and the `a` tag and replace them with the recommended `button`. 


if you take the second option of using a button, then you will have your CSS like this 👇 
```
.action:hover {
  color: orange;
  background-color: #396df2;
}
```

Reply:
Please do the same for all other buttons.